Source/relay/TourGuide/Items of the Month/2022/Autumnaton.ash

```diff
@@ -71,33 +71,33 @@ void IOTMAutumnatonGenerateTasks(ChecklistEntry [int] task_entries, ChecklistEnt
 	
 	if (__misc_state["in run"] && my_path().id != 25)
 	{
-		if (locationAvailable($location[sonofa beach]) == true && available_amount($item[barrel of gunpowder]) < 5)
+		if (locationAvailable($location[sonofa beach]) == true && get_property("sidequestLighthouseCompleted") == "none" && available_amount($item[barrel of gunpowder]) < 5)
 		{
-			targets.listAppend("barrel of gunpowder");
+			targets.listAppend("barrel of gunpowder (Sonofa Beach)");
 		}
 		if (locationAvailable($location[twin peak]) == false && get_property_int("chasmBridgeProgress") < 30)
 		{
-			targets.listAppend("bridge parts");
+			targets.listAppend("bridge parts (The Smut Orc Logging Camp)");
 		}
-		if (get_property_int("hiddenBowlingAlleyProgress") < 6)
+		if (get_property_int("hiddenBowlingAlleyProgress") + available_amount($item[bowling ball]) < 6)
```

Comment:
hm. does this work? in the quest tile `(get_property_int("hiddenBowlingAlleyProgress") >= 8);` is how it shows quest completion. does 6 work here or do we need 7/8 lol

Reply:
I won't lie, I was trusting the code that was here lol.

https://wiki.kolmafia.us/index.php/Quest_Tracking_Preferences#hiddenBowlingAlleyProgress

Based on this I think it's right though! It looks like 6 is when you encounter the protector spirit.